webcompat/static/js/lib/bugform.js

```diff
@@ -485,7 +492,7 @@ function BugForm() {
       // grab the data URI from background-image property, since it's already
       // in the DOM: 'url("data:image/....")'
       this.previewEl.get(0).style.backgroundImage.slice(5, -2),
-      function(response) {
+      _.bind(function(response) {
```

Comment:
Why are we adding _.bind here? I think I just removed it during a recent review.